### PR TITLE
Style sign petition

### PR DESCRIPTION
**Pull request by alexander-bobin**
@otlaitil and @alexander-bobin pair

You can ignore the commit bfc268c

https://www.pivotaltracker.com/story/show/95656562

### CODE REVIEW

File: app/assets/stylesheets/petitions/_icons.scss [hunk 1]
```diff
@@ -122,43 +122,43 @@
...
+    background-image: url("/images/icons/icon_share-twitter-2x.png");
     background-size: 100%;
   }
 }
-.icon-whatsapp-share {
+.icon-share-whats-app {
   width: 19px;
   height: 19px;
-  background-image: url("/images/icons/icon_whatsapp-share.png");
+  background-image: url("/images/icons/icon_share-whats-app.png");
```
Comment (h-lame) [inline app/assets/stylesheets/petitions/_icons.scss]: The image icons are actually called ``icon_share-whatsapp.png`` and ``icon_share-whatsapp-2x.png``, a greedy find-n-replace perhaps?

Comment (alexander-bobin) [inline app/assets/stylesheets/petitions/_icons.scss]: An erroneous find and replace. Will fix

File: app/helpers/form_helper.rb [hunk 2]
```diff
@@ -237,9 +237,9 @@ def countries_for_select
 
   def error_messages_for_field(object, field_name, opts={})
     unless object.errors[field_name].empty?
-      return_string = "<div class=\"errors#{' server_only_validation' if opts[:server_only_validation]}\">"
+      return_string = "<span class=\"error-message#{' server_only_validation' if opts[:server_only_validation]}\">"
       return_string += object.errors[field_name].join " "
-      raw(return_string + '</div>')
+      raw(return_string + '</span>')
```
Comment (pixeltrix) [inline app/helpers/form_helper.rb]: This method can be simplified as:

``` ruby
def error_messages_for_field(object, field_name, options = {})
  if errors = object.errors[field_name].presence
    content_tag :span, errors.join(' and '), { class: 'error-message' }.merge(options)
  end
end
```

File: app/views/petitions/show.html.erb [hunk 3]
```diff
@@ -63,20 +63,11 @@
 
 <p class="signature-count"><%= number_with_delimiter(@petition.signature_count) %> <span>signatures</span></p>
 
-<%= render 'shared/about_petitions' %>
+<%= render partial: 'shared/about_petitions', locals: { variation: "about-petitions-light" } %>
```
Comment (pixeltrix) [inline app/views/petitions/show.html.erb]: The default for `render` is now `:partial` so you can just use the following:

``` erb
<%= render 'shared/about_petitions', variation: 'about-petitions-light' %>
```

As you can see, there's no need to wrap the locals as `locals: {}`

File: app/views/petitions/show.html.erb [hunk 4]
```diff
@@ -63,20 +63,11 @@
...
-  <% title = @petition.title + " - e-petitions" -%>
-  <ul class="petition-share">
-    <li>
-      <%= link_to "Facebook", "http://www.facebook.com/sharer.php?t=#{URI.escape(title)}&u=#{URI.escape(petition_url(@petition))}" %>
-    </li>
-    <li>
-      <%= link_to "Twitter", "http://twitter.com/share?text=#{URI.escape(title)}&url=#{URI.escape(petition_url(@petition))}" %>
-    </li>
-  </ul>
+  <%= render partial: 'shared/share_petition', locals: { petition: @petition } %>
```
Comment (pixeltrix) [inline app/views/petitions/show.html.erb]: [Ditto][1]

[1]: https://github.com/alphagov/e-petitions/pull/106/files#r31999048

**Comment by pixeltrix** [comment]
Other than the comments I've posted, LGTM

File: app/helpers/form_helper.rb (hunk 2 above)
Comment (alexander-bobin) [inline app/helpers/form_helper.rb]: Will do

File: app/views/petitions/show.html.erb (hunk 3 above)
Comment (alexander-bobin) [inline app/views/petitions/show.html.erb]: Will do

**Pull request merged**